src/containers/shared/components/Transaction/AMMVote/Simple.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { SimpleRow } from '../SimpleRow'
+import { TransactionSimpleProps } from '../types'
+import { formatTradingFee } from '../../../utils'
+import { Account } from '../../Account'
+import { Amount } from '../../Amount'
+
+export const Simple = ({ data }: TransactionSimpleProps) => {
+  const { t } = useTranslation()
+  const { amount, amount2, fee, tradingFee, ammAccountID } = data.instructions
+  const value =
+    amount?.amount && amount.amount !== fee ? (
```

Comment:
ditto

Reply:
Even in single asset deposits XRP is included as an asset in the TX. When we look it up in the metadata we find that its only a fee, so we filter it out. Therefore this code needs to be included. Take a look at the json files for single asset deposit.